### DISCUSSION TITLE
How to correctly integrate AIR splash images into assets.car?

### DISCUSSION THREAD

**Discussion by GeneralVimes** [airsdk/Adobe-Runtime-Support#1874] [Q&A] [answered]
As I see in the newest release notes, if the user of free AIR tier uses Assets.car, HARMAN splash images won't be shown automatically at the startup. Instead it's needed to integrate the AIR splash images into Assets.car. So here is my question

I looked at https://airnativeextensions.github.io/tutorials/ios-icons-assets-car tutorial by @distriqt-developer . The Method 1 worked really well as it is. But I'm not sure where to add the Harman's screens (which are in “lib/aot/res” SDK folder).

The Distriqt's folder for making Assets.car is called Assets.xcassets and has 2 subfolders: **AppIcon.appiconset** and **LaunchImage.imageset**

The “lib/aot/res” SDK folder has 2 subfolders: **splash_landscape.imageset** and **splash_portrait.imageset**

I tried 2 ways: 
1) just copy folders to  **splash_landscape.imageset** and **splash_portrait.imageset** into Assets.xcassets and place them alongside with **AppIcon.appiconset** and **LaunchImage.imageset**

2) Replacing the contents of **LaunchImage.imageset** folder with the contents of **splash_portrait.imageset** folder.

Both times Assets.car war created successfully. But when I add it to a project and build an .ipa (Ad-Hoc), I don't see Harman splash, just a white screen splash at the start of the game.

Could you help me please, how exactly should I add Harman splashes into Assets.car? What am I missing?

**Comment by ajwfrost** [comment, marked as answer]
The code will be looking for an imageset called "splash_portrait" (or if the app is set for landscape orientation, "splash_landscape") - so it sounds like you're doing it right when using your first method. Are you able to check that they are present with this name within the resulting .car file?

thanks

**Reply by GeneralVimes** [reply]
Thank you! I've just redone the first method again carefully and now it is working (I suppose previous time time I messed something inside the folder).

So, for other developers who might need this, I just took assets-car-build folder from https://airnativeextensions.github.io/tutorials/ios-icons-assets-car and inside its Assets.xcassets there are 4 subfolders:
AppIcon.appiconset
LaunchImage.imageset
splash_portrait.imageset
splash_landscape.imageset

The first two originally were there, and the other two I copied from “lib/aot/res” SDK folder

The splash screen was shown well.

**Reply by marchbold** [reply]
Probably should update our scripts and tutorials to include these resources. 

@ajwfrost Can you confirm those two imagesets are the only thing needed in the assets car for the AIR SDK?

**Reply by ajwfrost** [reply]
Yes if you're using our "free tier" launch screen then it just looks for those splash_portrait or splash_landscape image sets...
thanks

**Reply by marchbold** [reply]
Just a quick note, I've updated the image scripts to include the AIR imagesets.

https://docs.airnativeextensions.com/docs/tutorials/ios-icons-assets-car/
